examples/local-strings.htm

```diff
@@ -22,6 +22,7 @@
         app:      'back-to-the-future',
         localization: {
           strings: {
+            allocatedResourcePrefix: 'Your Guide',
```

Comment:
Your guide? is that the default?

Reply:
How will this look for someone that does not change anything  after we deploy?

Reply:
no, the default is in the 'defaults' file. the default being "with". This is just an example, and that is why I have chosen to change the text to something different.

Reply:
makes semse